libraries/teams-scenarios/messagingExtensionConfig/src/messagingExtensionConfigBot.ts

```diff
@@ -14,7 +14,16 @@ import {
     ActionTypes,
 } from 'botframework-schema'
 
+/**
+ * After uploading the manifest you can click the dots in the extension menu at the bottom, or search for the
+ * exntesion in the command bar. From the extension window or the command bar you can click on the 3 dots on the specific extension to trigger
+ * the OnMessageActivityAsync function. If you click on the "Settings" tab you will fire the OnTeamsMessagingExtensionConfigurationSettingAsync
```

Comment:
Are these methods named like this in JS?

Reply:
Fixed, thanks